code/game/objects/structures/aliens.dm

```diff
@@ -1,428 +1,428 @@
-
-/obj/structure/alien/egg/attack_hand(mob/living/user)
-	if(user.get_int_organ(/obj/item/organ/internal/xenos/plasmavessel))
-		switch(status)
-			if(BURST)
-				to_chat(user, "<span class='notice'>You clear the hatched egg.</span>")
-				playsound(loc, 'sound/effects/attackblob.ogg', 100, 1)
-				qdel(src)
-				return
-			if(GROWING)
-				to_chat(user, "<span class='notice'>The child is not developed yet.</span>")
-				return
-			if(GROWN)
-				to_chat(user, "<span class='notice'>You retrieve the child.</span>")
-				Burst(0)
-				return
-	else
-		to_chat(user, "<span class='notice'>It feels slimy.</span>")
-		user.changeNext_move(CLICK_CD_MELEE)
-
-
-/obj/structure/alien/egg/proc/GetFacehugger()
-	return locate(/obj/item/clothing/mask/facehugger) in contents
-
-/obj/structure/alien/egg/proc/Grow()
-	icon_state = "egg"
-	status = GROWN
-
-/obj/structure/alien/egg/proc/Burst(kill = 1)	//drops and kills the hugger if any is remaining
-	if(status == GROWN || status == GROWING)
-		icon_state = "egg_hatched"
-		flick("egg_opening", src)
-		status = BURSTING
-		spawn(15)
-			status = BURST
-			var/obj/item/clothing/mask/facehugger/child = GetFacehugger()
-			if(child)
-				child.loc = get_turf(src)
-				if(kill && istype(child))
-					child.Die()
-				else
-					for(var/mob/M in range(1,src))
-						if(CanHug(M))
-							child.Attach(M)
-							break
-
-/obj/structure/alien/egg/bullet_act(obj/item/projectile/Proj)
-	if((Proj.damage_type == BRUTE || Proj.damage_type == BURN))
-		health -= Proj.damage
-	..()
-	healthcheck()
-
-
-/obj/structure/alien/egg/attackby(obj/item/I, mob/user, params)
-	if(I.attack_verb.len)
-		visible_message("<span class='danger'>[user] has [pick(I.attack_verb)] [src] with [I]!</span>")
-	else
-		visible_message("<span class='danger'>[user] has attacked [src] with [I]!</span>")
-
-	var/damage = I.force / 4
-	if(istype(I, /obj/item/weldingtool))
-		var/obj/item/weldingtool/WT = I
-
-		if(WT.remove_fuel(0, user))
-			damage = 15
-			playsound(loc, WT.usesound, 100, 1)
-
-	health -= damage
-	user.changeNext_move(CLICK_CD_MELEE)
-	healthcheck()
-
-
-/obj/structure/alien/egg/proc/healthcheck()
-	if(health <= 0)
-		if(status != BURST && status != BURSTING)
-			Burst()
-		else if(status == BURST && prob(50))
-			qdel(src)	//Remove the egg after it has been hit after bursting.
-
-
-/obj/structure/alien/egg/temperature_expose(datum/gas_mixture/air, exposed_temperature, exposed_volume)
-	if(exposed_temperature > 500)
-		health -= 5
-		healthcheck()
-
-
-/obj/structure/alien/egg/HasProximity(atom/movable/AM)
-	if(status == GROWN)
-		if(!CanHug(AM))
-			return
-
-		var/mob/living/carbon/C = AM
-		if(C.stat == CONSCIOUS && C.get_int_organ(/obj/item/organ/internal/body_egg/alien_embryo))
-			return
-
-		Burst(0)
-
-#undef BURST
-#undef BURSTING
-#undef GROWING
-#undef GROWN
-#undef MIN_GROWTH_TIME
-#undef MAX_GROWTH_TIME
-
-#undef WEED_NORTH_EDGING
-#undef WEED_SOUTH_EDGING
-#undef WEED_EAST_EDGING
-#undef WEED_WEST_EDGING
+/* Alien shit!
```

Comment:
This file seems to have randomly decided to show up in my PR after the merge conflict fix.

I did not change any of this file.